{"name": "Авторизация с указанием некоррекных данных", "status": "passed", "description": "Проверка  ошибки авторизации пользователя с некоррекными данными", "start": 1715557191966, "stop": 1715557192467, "uuid": "a483f590-5a01-42f2-9b71-3330d23064eb", "historyId": "1855a2b34a27c0230986758200d2022d", "testCaseId": "1855a2b34a27c0230986758200d2022d", "fullName": "tests.auth.test_login_user.TestLoginUser#test_user_authorization_with_incorrect_data", "labels": [{"name": "parentSuite", "value": "tests.auth"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "LAPTOP-0Q6BABTN"}, {"name": "thread", "value": "22552-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.auth.test_login_user"}]}